pkg/base/testdata/raft_config

```diff
@@ -12,6 +12,7 @@ echo
  RaftMaxSizePerMsg: (uint64) 32768,
  RaftMaxCommittedSizePerReady: (uint64) 67108864,
  RaftMaxInflightMsgs: (int) 128,
+ RaftMaxInflightBytes: (uint64) 268435456,
  RaftDelaySplitToSuppressSnapshotTicks: (int) 230
 }
 RaftHeartbeatInterval: 1s
```

Comment:
Could you add a few test cases to this that have a wildly inflated MaxInflightBytes and/or MaxInflightMsgs? Maybe not worth it as long as MaxInflightBytes is a very large constant value but if you're making it any more dynamic than that some more coverage of the configurations some users run in ("just disable MaxInflightBytes") would be good.

Reply:
Done.